src/java.desktop/windows/native/libawt/windows/awt_DnDDT.cpp

```diff
@@ -27,11 +27,12 @@
 #include <shellapi.h>
 #include <memory.h>
 
+// Don't want to pull in the redefined allocation functions
+#include "awt_ole.h"
```

Comment:
```suggestion
// awt_ole.h must be included before awt.h, since the latter redefines malloc 
// to Do_Not_Use_Malloc, etc, and that will break awt_ole.h.
#include "awt_ole.h"

```